content/en/logs/log_collection/csharp.md

```diff
@@ -24,11 +24,21 @@ further_reading:
   text: "Log Collection Troubleshooting Guide"
 ---
 
-To send your C# logs to Datadog, log to a file and then tail that file with your Datadog Agent. This page details setup examples for the `Serilog`, `NLog`, and `log4net` logging libraries.
+To send your C# logs to Datadog, use one of the following approaches:
+
+- Log to a file and then tail that file with your Datadog Agent. 
```

Comment:
These 3 list items should have links to their corresponding sections in the page

Reply:
Added in https://github.com/DataDog/documentation/pull/13907/commits/1a70bc07eeaf48a2d0e0d514ac2e270405e25364